testpar/t_pflush1.c

```diff
@@ -215,6 +215,7 @@ main(int argc, char *argv[])
     HDfflush(stderr);
     HDprintf("*** ERROR ***\n");
     HDprintf("THERE WAS A REAL ERROR IN t_pflush1.\n");
+    HDfflush(stdout);
```

Comment:
No. 10 This call to HDfflush(stdout); is not in the develop branch.   Why is it being added here?

Reply:
I didn't compare to dev, I think 1.12.